frontend/lib/src/components/shared/BaseColorPicker/BaseColorPicker.tsx

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 
-import React from "react"
+import React, { memo } from "react"
 
 import { StatefulPopover as UIPopover } from "baseui/popover"
 import { ChromePicker, ColorResult } from "react-color"
+// @ts-ignore
+import Saturation from "react-color/es/components/common/Saturation"
```

Comment:
Typescript complained about this import and I couldn't figure out how to fix it, so I added ts-ignore. Not sure if there's a better way to do this...

Reply:
Hm apparently the tests fail because of this, @lukasmasuch any idea how to fix this?